controllers/hbase_controller.go

```diff
@@ -644,6 +644,24 @@ func (r *HBaseReconciler) statefulSet(hb *hbasev1.HBase,
 	stsName, cmName types.NamespacedName, ss hbasev1.ServerSpec) (*appsv1.StatefulSet, string) {
 	spec := (&ss.PodSpec).DeepCopy()
 	spec.Volumes = append(spec.Volumes, configMapVolume(cmName))
+
+	templateMetadataAnnotations := cloneMap(ss.Metadata.Annotations, hb.Annotations)
+	ignoreAnnotations := []string{
```

Comment:
what about making it global and a map right away:

```
ignoredAnnotations = map[string]struct{}{
    "kubectl.kubernetes.io/last-applied-configuration": struct{}{},
}
```

Reply:
Done